### ISSUE TITLE
Unable to connect to the remote server

### ISSUE THREAD

**Issue by Manishjodhani** [haraka/Haraka#2386]
### system info

Haraka | Haraka.js — Version: 2.8.18
Node | v9.11.1
OS | Centos 6.9
### Expected behavior
![screenshot_23](https://user-images.githubusercontent.com/28861157/38659254-5ea4fbba-3e46-11e8-95e3-2692a8c16a7f.png)

Here the screen shot of error 
help me out?

**Comment by Dexus** [comment]
i would say you have no auth enabled or tls enabled. 
And for better help a LOG_PROTOCOL Debug would help.

**Comment by Manishjodhani** [comment]
Here again same problem
Please check this out
![screenshot_24](https://user-images.githubusercontent.com/28861157/38677097-dd35c7f0-3e7a-11e8-8ca8-248dfdfd0728.png)

**Comment by Dexus** [comment]
set ter log level to LOG_PROTOCOL! and post your server logs...

**Comment by baudehlo** [comment]
You haven't setup the server properly to receive mail for the domain you're
trying to deliver mail to.

There's several ways to do that. See this section of config/plugins:

# RCPT TO

# At least one rcpt_to plugin is REQUIRED for inbound email. The simplest

# plugin is in_host_list, see 'haraka -h rcpt_to.in_host_list' to configure.

rcpt_to.in_host_list

#qmail-deliverable

#rcpt_to.ldap

#rcpt_to.routes

**Comment by baudehlo** [comment]
Clearly you still haven't set it up right. Add mail.org to your
config/host_list file.

**Comment by Manishjodhani** [comment]
I have done that. Then it showing this error.
# haraka -c /etc/haraka/
loaded TLD files: 1=1544 2=8347 3=2424
loaded 8447 Public Suffixes
loglevel: INFO
log format: DEFAULT
Starting up Haraka version 2.8.18
[INFO] [-] [core] Loading plugins
[INFO] [-] [core] Loading plugin: tls
[INFO] [-] [core] loading tls.ini
[INFO] [-] [core] Loading plugin: auth/flat_file
[INFO] [-] [core] Loading plugin: rcpt_to.in_host_list
[NOTICE] [-] [core] Listening on IP:587
[NOTICE] [-] [core] Listening on IP:25
[NOTICE] [-] [core] Listening on IP:2525
[INFO] [-] [core] [outbound] Loading outbound queue from /etc/haraka/queue
[NOTICE] [8591E9E1-2333-4FCD-990F-FCBCE82CC460] [core] connect ip=178.145.245.45 port=53823 local_ip=IP local_port=25
[NOTICE] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core] sender <sender@mail.org> code=CONT msg=""
[INFO] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core]  hook=rcpt plugin=rcpt_to.in_host_list function=hook_rcpt params=<recivier@mail.org> retval=OK msg=""
[NOTICE] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core] recipient <recivier@mail.org> code=OK msg="" sender=info@rccess.org
[NOTICE] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core] message mid="" size=601 rcpts=1/0/0 delay=0 code=CONT msg=""
[NOTICE] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core] queue code=CONT msg=" (8591E9E1-2333-4FCD-990F-FCBCE82CC460.1)"
[INFO] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core] client [178.145.245.45] half closed connection
[NOTICE] [8591E9E1-2333-4FCD-990F-FCBCE82CC460.1] [core] disconnect ip=178.145.245.45 rdns=ns513574.ip.net helo=SkyWeb relay=N early=N esmtp=Y tls=N pipe=N errors=0 txns=1 rcpts=1/0/0 msgs=0/1/0 bytes=601 lr="451  (8591E9E1-2333-4FCD-990F-FCBCE82CC460.1)" time=1.326

**Comment by smfreegard** [comment]
Based on the screenshot - you're trying to use Haraka as an SMTP relay using SMTP AUTH, so you're going to need to set-up *and configure* the following plugins only:

tls (required for SMTP AUTH to work)
auth/flat_file  (required for SMTP AUTH; you will need to configure it with username/passwords)

And that is all - if you configure both of those by reading their documentation, then someone logging into Haraka using those SMTP AUTH credentials will be allowed to relay messages out to the internet.

You don't need rcpt_to.in_host_list in the case,as that it for inbound mail that you want to deliver to a specific destination.

**Comment by Manishjodhani** [comment]
Please check Once again Here is the log.
haraka -c /etc/haraka/
loaded TLD files: 1=1544 2=8347 3=2424
loaded 8447 Public Suffixes
loglevel: INFO
log format: DEFAULT
Starting up Haraka version 2.8.18
[INFO] [-] [core] Loading plugins
[INFO] [-] [core] Loading plugin: auth/flat_file
[INFO] [-] [core] Loading plugin: rcpt_to.in_host_list
[NOTICE] [-] [core] Listening on IP:587
[NOTICE] [-] [core] Listening on IP:25
[NOTICE] [-] [core] Listening on IP:2525
[INFO] [-] [core] [outbound] Loading outbound queue from /etc/haraka/queue
[NOTICE] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612] [core] connect ip=192.99.9.142 port=55885 local_ip=IP local_port=25
[NOTICE] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core] sender <sender@mail.com> code=CONT msg=""
[INFO] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core]  hook=rcpt plugin=rcpt_to.in_host_list function=hook_rcpt params=<recivier@mail.com> retval=OK msg=""
[NOTICE] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core] recipient <recivier@mail.com> code=OK msg="" sender=sender@mail.com
[NOTICE] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core] message mid="" size=623 rcpts=1/0/0 delay=0 code=CONT msg=""
[NOTICE] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core] queue code=CONT msg=" (F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1)"
[INFO] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core] client [192.99.9.142] half closed connection
[NOTICE] [F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1] [core] disconnect ip=192.99.9.142 rdns=ns513574.ip-192-99-9.net helo=SkyWeb relay=N early=N esmtp=Y tls=N pipe=N errors=0 txns=1 rcpts=1/0/0 msgs=0/1/0 bytes=623 lr="451  (F9F0D20D-C9DB-41D9-9E48-4EDE6A469612.1)" time=1.321
^C[NOTICE] [-] [core] SIGINT received
[INFO] [-] [core] Shutting down.

**Comment by smfreegard** [comment]
**NO TLS PLUGIN**

**Comment by Manishjodhani** [comment]
Please give the solution??

**Comment by KingNoosh** [comment]
How can we give you a solution when you don't understand what was said
previously within the thread?

Set the log level to protocol.

Rerun Haraka with your test case. Give paste the log here.

We're trying to help you but this is not a one sided process. It requires
work and cooperation on your side too.

**Comment by KingNoosh** [comment]
Also do you need SMTP AUTH? Is my first question as it doesn't seem like
you know what you're doing.

**Comment by smfreegard** [comment]
Yes - he does need SMTP AUTH.   Look at the screen-shot he posted.   I've repeatedly said that he need to enable TLS and he hasn't done so, which is why it isn't working.
